Scrabble/PlayerClass/Player.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+
+public class Player
+{
+    // Player class member variables
+    private string name;
+    private LetterRack letterRack;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/15116348/10874510/11531522-80e3-11e5-82c8-c0a6bf5a3c63.png)
Jenkins failed here. The error is because we haven't created the LetterRack class yet.